plugin/helpers.go

```diff
@@ -12,17 +11,27 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/Benchkram/errz"
+	"github.com/puppetlabs/wash/journal"
 	log "github.com/sirupsen/logrus"
 )
 
 // DefaultTimeout is the default timeout for prefetching
 var DefaultTimeout = 10 * time.Second
 
+/*
+Name returns the entry's name as it was passed into
+plugin.NewEntry. It is meant to be called by other
+Wash packages. Plugin authors should use EntryBase#Name
```

Comment:
I'm still not sure why both of these are needed.

Reply:
This is needed by the API, or anything that needs the entry's name. That's b/c `Name` isn't part of the entry interface. Can clarify it a bit more.